Samples/Caboodle.Samples/ViewModel/EmailViewModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Caboodle;
+using Xamarin.Forms;
+
+namespace Caboodle.Samples.ViewModel
+{
+    public class EmailViewModel : BaseViewModel
+    {
+        string tomails;
+        string ccmails;
+
+        public EmailViewModel()
+        {
+            // For testing mailboxes (w/o registration) go to:
+            // https://www.mailinator.com
+            tomails = string.Join(",", new string[] { "caboodle@mailinator.com", "caboodle01@mailinator.com" });
+            ccmails = string.Join(";", new string[] { "caboodle@mailinator.com", "caboodle02@mailinator.com" });
```

Comment:
I don't think we should use third party tools - maybe there is a testing tool that just accepts everything - maybe an example.org type of thing?